doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc

```diff
@@ -15,17 +15,25 @@ Navigate to the {PRODUCT} {PRODUCT_VERSION} `bin` directory and start the new {K
 ----
 $ ~/EAP_HOME/bin/standalone.sh --server-config=standalone-full.xml <1>
 -Dorg.kie.server.id=myserver <2>
--Dorg.kie.server.controller=http://localhost:8080/{URL_COMPONENT_CENTRAL}/rest/controller <3>
--Dorg.kie.server.location=http://localhost:8080/kie-server/services/rest/server <4>
--Dorg.kie.server.persistence.dialect=org.hibernate.dialect.PostgreSQLDialect <5>
--Dorg.kie.server.persistence.ds=java:jboss/datasources/psjbpmDS <6>
+-Dorg.kie.server.user=username <3>
+-Dorg.kie.server.pwd=password <4>
+-Dorg.kie.server.controller=http://localhost:8080/{URL_COMPONENT_CENTRAL}/rest/controller <5>
+-Dorg.kie.server.controller.user=process_server_username <6>
+-Dorg.kie.server.controller.pwd=process_server_password <7>
```

Comment:
This should be something like controller_username and password. And process_server_username should be for org.kie.server.user parameter.

Reply:
Changed as suggested.